ds3_java_cli/src/main/java/com/spectralogic/ds3cli/util/Utils.java

```diff
@@ -144,6 +146,8 @@ private static String unixNormalizeObjectName(final String objectName) {
             path = objectName.substring(1);
         } else if (objectName.startsWith("./")) {
             path = objectName.substring(2);
+        } else if (objectName.startsWith("../")) {
+            path = objectName.substring(3);
         } else {
             path = objectName;
         }
```

Comment:
should this be wrapped in some sort of a loop, which breaks when path == objectName?  I'm thinking of the ../../somethingSomething case